builder/docker.go

```diff
@@ -69,58 +68,56 @@ func loadDockerManifest(ctx context.Context, console conslogging.ConsoleLogger,
 		"%s is a multi-platform image. The following per-platform images have been produced:\n\t%s\n%s\n",
 		parentImageName, strings.Join(childImgs, "\n\t"), noteDetail)
 
-	cmd := exec.CommandContext(ctx, "docker", "tag", children[defaultChild].imageName, parentImageName)
-	cmd.Stdout = os.Stderr // Preserve desired output on stdout, all logs to stderr
-	cmd.Stderr = os.Stderr
-	err := cmd.Run()
+	err := fe.ImageTag(ctx, containerutil.ImageTag{
+		SourceRef: children[defaultChild].imageName,
+		TargetRef: parentImageName,
+	})
 	if err != nil {
 		return errors.Wrap(err, "docker tag default platform image")
 	}
 	return nil
 }
 
-func loadDockerTar(ctx context.Context, r io.ReadCloser, console conslogging.ConsoleLogger) error {
-	cmd := exec.CommandContext(ctx, "docker", "load")
-	cmd.Stdin = r
-	output, err := cmd.CombinedOutput()
+func loadDockerTar(ctx context.Context, fe containerutil.ContainerFrontend, r io.ReadCloser, console conslogging.ConsoleLogger) error {
+	err := fe.ImageLoad(ctx, r)
 	if err != nil {
-		console.Warnf("%+v output:\n%s\n", cmd.Args, string(output))
-		return errors.Wrapf(err, "docker load")
+		console.Warnf(err.Error())
```

Comment:
I think this is no longer necessary. We used to have this in order to also get the output itself (otherwise the error alone would say just something along the lines of "exited with non-zero").

The wrapped error is probably reported somewhere upstream anyway.